pkg/update/update_cockroach_version.go

```diff
@@ -208,8 +208,8 @@ func kindAndCheckPreserveDowngradeSetting(
 		// To do a roll forward, preserve downgrade option should either be
 		// unset, or set to current version. If unset, kubeupdate will set it to
 		// current version.
-		if preserve.Compare(&semver.Version{}) != 0 &&
-			(preserve.Major() != currentVersion.Major() || preserve.Minor() != currentVersion.Minor()) {
+		if  (preserve.Compare(&semver.Version{}) != 0 &&
```

Comment:
@chrisseto need your feedback here

Reply:
this is the same, only the format is different